### DISCUSSION TITLE
[@types/jest]: error TS2694: Namespace 'NodeJS' has no exported member 'Global'.

### DISCUSSION THREAD

**Discussion by splincode** [DefinitelyTyped/DefinitelyTyped#54305] [Issues with a @types package]
Released https://jestjs.io/blog/2021/05/25/jest-27

But typings not working for jest-27

**Comment by mrcnski** [comment]
I have braised the issue [here](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/54294) and provided some more info.
